src/components/m-dropdown/index.scss

```diff
@@ -28,7 +28,6 @@
 
 .m-dropdown__select {
   @include dropdown-select();
-  @include typo-text-longer();
```

Comment:
Please review this with design/UX, typography is mandatory

Reply:
I removed all datepicker changes. they were not meant to be in this PR.

Reply:
Okay, though I still see the deletions here

Reply:
As discussed I pushed again with an interactive rebase without the unwished changes.